src/cursor/cur_file.c

```diff
@@ -1097,6 +1097,54 @@ __curfile_create(WT_SESSION_IMPL *session, WT_CURSOR *owner, const char *cfg[],
     return (ret);
 }
 
+/*
+ * __wti_cursor_largest_key --
+ *     WT_CURSOR->largest_key default implementation..
+ */
+int
+__wti_cursor_largest_key(WT_CURSOR *cursor)
```

Comment:
Why did you move this function? It's a generic implementation that's not specific to file or even btree cursors. It's still in `src/cursor/cur_std.c` on develop.

Reply:
It needs to use the WT_WITH_CHECKPOINT macro